.github/tests/test_script.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/python
+
+# Copyright (C) 2020 Vanessa Sochat.
+
+# This Source Code Form is subject to the terms of the
+# Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed
+# with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from jsonschema import validate
+from jsonschema.exceptions import ValidationError
+
+import json
+import os
+import re
+import pytest
+import yaml
+
+here = os.path.dirname(os.path.abspath(__file__))
+root = os.path.dirname(os.path.dirname(here))
+
+
+def load_schema(path):
+    """load a schema from file. We assume a json file
+    """
+    with open(path, "r") as fd:
+        schema = json.loads(fd.read())
+    return schema
+
+
+def load_recipe(path):
```

Comment:
Same thing i think this method `load_recipe` can be a imported by all schema test assuming it is defined somewhere lets say under ``utils/schema.py`` and we can import as ``from utils/schema import load_recipe``

Reply:
Yep, you are spot on I'll have a lot of this functionality implemented in buildtest. Just need these schemas to work with first!